src/context/WalletProvider/NativeWallet/components/LegacyLogin.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  Button,
+  Checkbox,
+  FormControl,
+  FormErrorMessage,
+  Input,
+  ModalBody,
+  ModalHeader,
+  useColorModeValue,
+} from '@chakra-ui/react'
+import { useState } from 'react'
+import { FieldValues, useForm } from 'react-hook-form'
+import { useTranslate } from 'react-polyglot'
+import { RouteComponentProps } from 'react-router-dom'
+import { Card } from 'components/Card/Card'
+import { Text } from 'components/Text'
+
+// @TODO(NeOMakinG): Remove this and add the 2fa code logic
+const twoFactorAuthCode = ''
+
+export const LegacyLogin = ({ history }: RouteComponentProps) => {
+  const [isCaptchaValidated, setCaptchaValidated] = useState(false)
```

Comment:
Might be my nitpicky, pedantic self commenting again but:
```suggestion
  const [isCaptchaSolved, setCaptchaSolved] = useState(false)
```
or:
```suggestion
  const [isCaptchaResolved, setCaptchaResolved] = useState(false)
```
with a preference for the first option.

This kind of terminology is important for grepping across (the) project(s) - someone searching for captcha resolution might grep for `solve` or `resolve`